src/tests/DataLayer.test.js

```diff
@@ -105,11 +105,7 @@ test('add event', () => {
 
 test('check dataLayer change event was executed', () => {
   const mockCallback = jest.fn();
-  const argOn = {
-    on: 'datalayer:change',
-    handler: mockCallback
-  };
-  dataLayer.push(argOn);
+  dataLayer.addEventListener('datalayer:change', mockCallback);
```

Comment:
If I'm not mistaken, there are more tests to refactor: at least all the tests that have `argOn` and `argOff`

Reply:
yes, on it now!